backend/src/main/java/gov/cdc/usds/simplereport/utils/DateTimeUtils.java

```diff
@@ -37,8 +37,33 @@ public class DateTimeUtils {
   private static final ZoneId hawaiiTimeZoneId = ZoneId.of("US/Hawaii");
   private static final ZoneId aleutianTimeZoneId = ZoneId.of("US/Aleutian");
   private static final ZoneId samoaTimeZoneId = ZoneId.of("US/Samoa");
+  private static final ZoneId puertoRicoTimeZoneId = ZoneId.of("America/Puerto_Rico");
 
-  public static final Map<String, ZoneId> validTimeZoneIdMap =
+  /**
+   * Map of common name `time_zone` values returned from SmartyStreets and their corresponding
+   * ZoneId https://www.smarty.com/docs/cloud/us-street-api
+   */
+  public static final Map<String, ZoneId> commonNameZoneIdMap =
```

Comment:
⚠️ explicitly mapped all the possible SmartyStreets `time_zone` common names that could be returned to specific `ZoneId`s

Reply:
strange that they use this list instead of something more standard